packages/editor/src/components/post-actions/actions.js

```diff
@@ -202,7 +313,7 @@ const trashPostAction = {
 						disabled={ isBusy }
 						__experimentalIsFocusable
 					>
-						{ __( 'Delete' ) }
+						{ __( 'Trash' ) }
```

Comment:
```suggestion
						{ __( 'OK' ) }
```

I think "OK" is easier to understand, just like in WP6.5, what do you think?

### WP6.5

![image](https://github.com/WordPress/gutenberg/assets/54422211/fa69adcf-c777-4cf1-bd22-55b1ef53ce2c)

### This PR

![image](https://github.com/WordPress/gutenberg/assets/54422211/8511ccf4-7205-4052-b5bd-24ab6359cf34)